src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -92,6 +100,15 @@ export const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputPro
     setValue("pictureUrl", pictureUrlRef, true);
   };
 
+  const githubImageSrc =
+    githubHandle && `https://github.com/${githubHandle}.png?size=120`;
+
+  useEffect(() => {
+    if (!githubImageSrc) return;
+
+    setValue("pictureUrl", githubImageSrc, true);
+  }, [githubImageSrc, setValue]);
```

Comment:
Keeping it together like this will make it easier to see the relation.

```suggestion
  useEffect(() => {    
    if (!githubHandle) return;

    const githubImageSrc =
    githubHandle && `https://github.com/${githubHandle}.png?size=120`;
    
    setValue("pictureUrl", githubImageSrc, true);
  }, [githubHandle, setValue]);
```

Reply:
Seemingly we use `githubImageSrc` later on. Though i'm confused why we need the `setValue` here then?

Reply:
@0xdevalias to make it the default picture

Reply:
Fair